packages/amplication-server/src/core/packages/dto/Package.ts

```diff
@@ -0,0 +1,40 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+import { IBlock } from "../../../models";
+import { EnumPackageStatus } from "../enums/EnumPackageStatus";
+import { PackageFile } from "./PackageFile";
+
+@ObjectType({
+  isAbstract: true,
+  implements: [IBlock],
+})
+export class Package extends IBlock {
+  @Field(() => String, {
+    nullable: false,
+  })
+  summary!: string;
+
+  @Field(() => EnumPackageStatus, {
+    nullable: false,
+  })
+  status: keyof typeof EnumPackageStatus;
+
```

Comment:
Let's remove everything under this line, just stay with the `summary` and the `status` (and the other things inherited from Block anyway)